week1/pages/add.js

```diff
@@ -0,0 +1,16 @@
+
+import HTTP from 'http'
+import STATUS  from './status.js'
+ 
+ 
+
+module.exports = {
+    "body" : function contentHomePage(Response ){
```

Comment:
I think you complicated yourself a bit with this design. You could just export `function contentHomePage(Response) { ... }` and it should still work right?